src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -332,6 +333,18 @@ protected function pExpr_Array(Array_ $array): string
         return parent::pExpr_Array($array);
     }
 
+    protected function pExpr_BinaryOp_Pipe(Pipe $node, int $precedence, int $lhsPrecedence): string
+    {
+        return $this->pInfixOp(
+            Pipe::class,
+            $node->left,
+            $this->nl . '    |> ',
```

Comment:
indent size is not always 4, this may more reliable with `resolveIndentSpaces()` method:

```suggestion
            "\n" . $this->resolveIndentSpaces() . '|>'
```